Tweaks/TweaksAssembly/TweaksAssembly.csproj

```diff
@@ -9,7 +9,7 @@
     <TargetFramework>net35</TargetFramework>
     <FileAlignment>512</FileAlignment>
     <LangVersion>7.3</LangVersion>
-    <CodeAnalysisRuleSet>TweaksAssembly.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSet></CodeAnalysisRuleSet>
```

Comment:
Could you not remove this?

Reply:
Fixed. I believed I removed this as it wasn't causing be me to build. Apologies. Though I ask to be pinged on discord (blckhawker) for messages that like these as it's very easy for them to be missed.

Reply:
Also if this was the only change, could you not revert it yourself? I understand KTANE is not your biggest priority, but if requesting a small change like this means the pr isn't going to be approved for 3 months, I think it would be more efficient for everyone if you made the change and just informed me for future PRs.